spec/app.spec.js

```diff
@@ -1,12 +1,22 @@
 // @flow
 import request from 'supertest';
 import app from '../src/app';
+import GetArtifactRequest from './Get_Artifacts_Request';
 
 describe('app', () => {
-  it('should respond with a 200', (done) => {
+  it('should respond with a 200 to GET', (done) => {
     request(app)
       .get('/')
       .expect(200)
       .end(done);
   });
 });
+describe('app', () => {
+  it('should respond with a 200 to GetArtifacts', (done) => {
```

Comment:
same as above extra describe blocks are usually pretty cool, especially since you will most likely end up with more than one `it` statement.